srv/components/system/storage/teardown/init.sls

```diff
@@ -22,8 +22,13 @@ include:
 {% endif %}
 
 Mount default OS swap device:
-  cmd.run:
-    - name: swapon -a || true
+  module.run:
+    - mount.swapon:
+      - name: /dev/dm-1
```

Comment:
@83bhp is correct.
This is unreliable.
```
[root@srvnode-1 ~]# swapon
NAME      TYPE       SIZE USED PRIO
/dev/dm-6 partition 12.8G   0B   -2
```
 We need to query and switch-off based on system data.

Reply:
@SomwanshiAnjali 
This is a better alternative that works:
```
# Disable system swap
{% set swap_devices = salt['mount.swaps']().keys() %}
Deactivate and unmount system SWAP:
  module.run:
    # Deactivate SWAP
    {% for swap_device in swap_devices %}
    - mount.swapoff:
      - name: {{ swap_device }}
    {% endfor %}
    # Remove SWAP from /etc/fstab
    - mount.rm_fstab:
      - name: swap
      - device: /dev/mapper/vg_sysvol-lv_swap
```

Result:
```
local:
    ----------
    Deactivate and unmount system SWAP:
        ----------
        module.run:
            |_
              ----------
              mount.swapoff:
                  |_
                    ----------
                    name:
                        /dev/dm-6
            |_
              ----------
              mount.rm_fstab:
                  |_
                    ----------
                    name:
                        swap
                  |_
                    ----------
                    device:
                        /dev/mapper/vg_sysvol-lv_swap
```

Reply:
@ypise 
Addressed, please check